code/xcvm/lib/core/src/gateway/config.rs

```diff
@@ -163,7 +163,8 @@ pub enum ConfigSubMsg {
 	ForceRemoveAsset { asset_id: AssetId },
 
 	/// instantiates default interpreter on behalf of user
-	ForceInstantiate { user_origin: Addr },
+	/// `salt` - human string, converted to hex or base64 depending on implementation
+	ForceInstantiate { user_origin: Addr, salt: Option<String> },
```

Comment:
Perhaps:

```suggestion
	ForceInstantiate {
		user_origin: Addr,
		#[serde(skip_serializing_if = "String::if_empty", default)]
		salt: String,
	},
```

This makes the code simpler since we don’t need to worry about `None`
values and the end effect is the same, i.e. lack of salt is
interpreted as empty value.

Reply:
Also, should salt be `Binary` instead?